doc/lightning-listpeerchannels.7.md

```diff
@@ -9,7 +9,7 @@ SYNOPSIS
 DESCRIPTION
 -----------
 
-The **listpeerchannels** RPC command returns data on channels of the network, with the possibility to filter the channels by node id.
+The **listpeerchannels** RPC command returns data on this node's channels, with the possibility to filter the channels by the id of a remote node.
```

Comment:
idk if this is clear, maybe it is. However, listpeerchannels return the list of channels inside the network divided by peers.

e.g: 
peer -> channels
peer -> channels